packages/tailwindcss/src/compat/legacy-utilities.ts

```diff
@@ -0,0 +1,28 @@
+import { decl } from '../ast'
+import type { DesignSystem } from '../design-system'
+
+export function registerLegacyUtilities(designSystem: DesignSystem) {
+  for (let [value, direction] of [
+    ['t', 'top'],
+    ['tr', 'top right'],
+    ['r', 'right'],
+    ['br', 'bottom right'],
+    ['b', 'bottom'],
+    ['bl', 'bottom left'],
+    ['l', 'left'],
+    ['tl', 'top left'],
+  ]) {
+    designSystem.utilities.static(`bg-gradient-to-${value}`, () => [
+      decl('--tw-gradient-position', `to ${direction} in oklch,`),
+      decl('background-image', `linear-gradient(var(--tw-gradient-stops))`),
+    ])
+  }
```

Comment:
These need to be removed from the utilities.ts file

Reply:
Good catch, removed!